pymatgen/io/ase.py

```diff
@@ -216,12 +217,19 @@ def get_structure(atoms: Atoms, cls: type[Structure] = Structure, **cls_kwargs)
         else:
             sel_dyn = None
 
+        # Atoms.info <---> Structure.properties (excluding properties["calc"])
+        properties = jsanitize(getattr(atoms, "info", {}))
```

Comment:
@janosh: Just wanted to point out one subtlety to you. Currently, we have the mapping: `Atoms.info <--> Structure.properties`, since they serve the same purpose between ASE and Pymtatgen, respectively. However, I `jsanitize`-d the `atoms.info` before assigning it to `.properties` to ensure the `Structure`/`Molecule` object can be (de)serialized since this is often an implicit assumption of `Structure`/`Molecule` objects but is not necessarily one for ASE `Atoms` objects. In practice, this means that there could be some scenarios where a user interconverts between ASE `Atoms` and Pymatgen `Structure`/`Molecule` with some loss of information; namely, if the `atoms.info` is modified by `jsanitize()` then it can't be restored in its original form. There might be a better solution than this, but I implemented it because oftentimes ASE will dump un-serializable class objects (namely, a `Spacegroup` class) into `atoms.info`, such as when reading in a CIF with symmetry flags.

Reply:
Thanks for highlighting! Seems like a good way to go about this for now. I'll keep this in mind if we run into any issues.